### ISSUE TITLE
Breadcrumbs: Hover link underline (DS6.v2.01)

### ISSUE THREAD

**Issue by abiyasa** [eBay/skin#556]
On Playbook, when you hover on breadcrumb link, the color turns blue with underline 

![image](https://user-images.githubusercontent.com/1072047/52642629-aad03c80-2edb-11e9-9920-327cae5f4cab.png)

However, on Skin (DS6), it's just turn the color to blue but without underline. 
It should be underlined just like defined on Playbook.

**Comment by ianmcburnie** [comment]
Thanks @abiyasa 

We are in the process of adding the specific component version numbers from playbook to our skin docs. It's often the case that a Skin module is a revision or two behind the playbook (they move faster than we do), but thanks for catching this! We will update accordingly.

**Comment by ianmcburnie** [comment]
Will be available in 7.1.0
